src/use-dropdown-menu.ts

```diff
@@ -24,6 +24,7 @@ interface DropdownMenuResponse {
 		ref: React.RefObject<HTMLAnchorElement>;
 	}[];
 	readonly isOpen: boolean;
+	readonly moveFocus: (itemIndex: number) => void;
 	readonly setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
I would swap these two lines so that we keep `isOpen` and `setIsOpen` (which are logically related) together.

```suggestion
	readonly setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
	readonly moveFocus: (itemIndex: number) => void;
```